### DISCUSSION TITLE
可以自定义文件的面包屑导航吗

### DISCUSSION THREAD

**Discussion by chaoyue1217** [opensumi/core#1982] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/18662948/202345916-b9139e22-4e3c-4441-97b9-f84ff719da63.png)
如上图，希望可以显示文件里的具体函数名等

**Comment by erha19** [comment, marked as answer]
@chaoyue1217 面包屑中的函数名显示依赖语言服务，你可以在插件市场下寻找是否有对应语言的语法服务插件 `xxx language ...`, 安装后会有相关信息展示。

目前暂不支持通过其他方式对面包屑导航进行自定义的逻辑。
